### ISSUE TITLE
BiglyBT downloads wrong architecture swt file (aarch64) 

### ISSUE THREAD

**Issue by radumamy** [BiglySoftware/BiglyBT#2034]
I have completed a fresh install of the BiglyBT on my aarch64 Linux system. The client GUI loaded fine on first launch and then it updated swt and restarted itself. 

Now I get the below error 

`SWT check failed with: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons:
        no swt-gtk-4934r5 in java.library.path: [/opt/biglybt]
        no swt-gtk in java.library.path: [/opt/biglybt]
        /home/radumamy/.swt/lib/linux/aarch64/libswt-gtk-4934r5.so: /home/radumamy/.swt/lib/linux/aarch64/libswt-gtk-4934r5.so: cannot open shared object file: No such file or directory (Possible cause: can't load AMD 64-bit .so on a AARCH64-bit platform)
        Can't load library: /home/radumamy/.swt/lib/linux/aarch64/libswt-gtk.so
        /home/radumamy/.swt/lib/linux/aarch64/libswt-gtk-4934r5.so: /home/radumamy/.swt/lib/linux/aarch64/libswt-gtk-4934r5.so: cannot open shared object file: No such file or directory (Possible cause: can't load AMD 64-bit .so on a AARCH64-bit platform)`

It appears it downloaded the wrong architecture for the swt file as shown in the screenshot below. 

* OS and version openSUSE Tumbleweed 20210223
* BiglyBT Version Number 2.6.0.0
* For *nix: Xfce 4.16

![Screenshot_20210309-131105~2](https://user-images.githubusercontent.com/63680669/110476768-45f76880-80da-11eb-9194-daf4364e1f6b.png)

**Comment by radumamy** [comment]
Workaround:

I have copied the `swt-aarch64.jar` from `/opt/biglybt/swt/`  into `/opt/biglybt/` and the client launches again.

Thank you.

**Comment by parg** [comment]
Hopefully fixed, thanks!

**Comment by parg** [comment]
Yup - if you want try the beta program (See the Help menu for an option to join), there is an SWT update in that and it would be good to have at least one person test it!

**Comment by radumamy** [comment]
Thanks @parg. I've joined Beta and successfully updated swt with it.

**Comment by parg** [comment]
Thanks for your help!
